lib/Infer/AliveDriver.cpp

```diff
@@ -530,12 +536,21 @@ std::string getUniqueName() {
 bool souper::AliveDriver::translateAndCache(const souper::Inst *I,
                                             IR::Function &F,
                                             Cache &ExprCache) {
+  if (souper::Inst::isOverflowIntrinsicSub(I->K)) {
```

Comment:
it seems odd to special-case this one instruction, what's going on here?

Reply:
we are special casing out the SAddO instruction as they are not real instructions. The selected function checks if the given instruction kind is any of those 'AddO' type.

Reply:
ok-- add a brief comment to this effect, that these are souper-internal things that we don't bother alive with

Reply:
done